resource-aggregate/config.yaml

```diff
@@ -0,0 +1,85 @@
+log:
+  debug: false
+apis:
+  grpc:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    authorization:
+      authority:
+      audience:
+      ownerClaim: sub
```

Comment:
this value is authorization service client related, it has nothing to do with authorization of the grpc endpoint of the resource aggregate. It's relevant for northbound interface, to AuthService.

Reply:
Authorization works with JWT token and fill context with OwnerID so in code you just acquire OwnerID from context and not from JWT token. It's more clean.